adagrams/game.py

```diff
@@ -1,11 +1,135 @@
+import random
+
 def draw_letters():
-    pass
+    # Letters to choose from
+    letter_pool = {
+        'A': 9,
+        'B': 2,
+        'C': 2,
+        'D': 4,
+        'E': 12,
+        'F': 2,
+        'G': 3,
+        'H': 2,
+        'I': 9,
+        'J': 1,
+        'K': 1,
+        'L': 4,
+        'M': 2,
+        'N': 6,
+        'O': 8,
+        'P': 2,
+        'Q': 1,
+        'R': 6,
+        'S': 4,
+        'T': 6,
+        'U': 4,
+        'V': 2,
+        'W': 2,
+        'X': 1,
+        'Y': 2,
+        'Z': 1
+    }
+
+    letters = [] # List for random letters
+
+    # Pick random letter from dictionary for list until we have enough.
+    while len(letters) < 10:
+        letter = random.choice(list(letter_pool))
```

Comment:
This line of code re-creates the letter pool list on each iteration of the while loop. Given that the list will have the same letters each time, we can create that list once before the while loop and save it in a variable that we can call here instead.